frontend/views/pages/Page.vue

```diff
@@ -18,11 +18,11 @@ div(:data-test='pageTestName' :class='$scopedSlots.sidebar ? "p-with-sidebar" :
   main.p-main(:class='mainClass')
     slot
 
-  nav.p-sidebar(
-    :role='L("Secondary")'
+  section.p-sidebar(
     v-if='$scopedSlots.sidebar'
     :class='{ "is-active": ephemeral.isActive }'
   )
+    i18n.sr-only(tag='h2') Page details
```

Comment:
Can you update our documentation using this as an example of when to use `h2` without also adding an `.is-title-X` and explaining why?